### DISCUSSION TITLE
Welcome to openQA Discussions!

### DISCUSSION THREAD

**Discussion by okurz** [os-autoinst/openQA#3631] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hanbo515330** [comment]
Hello, May I ask how to install openQA offline?

**Reply by okurz** [reply]
> Hello, May I ask how to install openQA offline?

Handled in https://github.com/os-autoinst/openQA/discussions/6814
